corehq/apps/users/templates/users/partials/bootstrap5/edit_role_modal.html

```diff
@@ -528,16 +531,16 @@ <h4 class="modal-title" data-bind="text: modalTitle"></h4>
               </div>
             {% endif %}
             {% if request|request_has_privilege:"CUSTOM_DOMAIN_ALERTS" %}
-            <div class="form-group">  {# todo B5: css-form-group #}
-              <label class="col-md-4 form-label">
+            <div class="row">
+              <label class="col-md-4 form-label fw-bold">
```

Comment:
Same comment on `<label>`s throughout this file being using as headings. `<h4>`?